virttest/utils_libvirtd.py

```diff
@@ -441,6 +443,17 @@ def deprecation_warning():
                 "libvirtd service.")
 
 
+def deprecation_function_waring(func):
```

Comment:
utils_libvirtd is a module to control libvirtd service. I don't think this decorator should be here, what do you think?

Reply:
OK, where should we put it?

Reply:
Hmm, it must be a lower-level module if you want to make it to be generic. Otherwise, we can leave it to libvirt.py I think. 
@dzhengfy do you have any suggestions?

BTW, @nanli1 I think we'd better provide a replacement of the deprecated function. 
Please have a look at 
https://github.com/avocado-framework/avocado-vt/blob/6e3d2eb2d8390d3d0cd8d20c94f7e28940fb6e46/virttest/utils_hotplug.py#L59
https://github.com/avocado-framework/avocado-vt/blob/6e3d2eb2d8390d3d0cd8d20c94f7e28940fb6e46/virttest/utils_test/libvirt.py#L2405-L2406

Reply:
Added commend
```

 This function is deprecated now.
    The recommended usage is below:
    Define a parameter in cfg like
        sec_dict = {"secret_ephemeral": "yes", "secret_private": "yes",
        "description": "desc", "usage": "volume"}
    Set up the device in python like
        sec_dict = eval(params.get('sec_dict', '{}'))
        sec_xml = secret_xml.SecretXML("no", "yes")
        sec_xml.setup_attrs(**sec_dict)

```